src/pudl/analysis/allocate_gen_fuel.py

```diff
@@ -1012,7 +1012,7 @@ def _allocate_unassociated_pm_records(
     eia_generators_connected = gen_assoc.loc[connected_mask].assign(
         capacity_mw_minus_one=lambda x: x.groupby(idx_minus_one)[
             "capacity_mw"
-        ].transform(sum),
+        ].transform("sum"),
```

Comment:
Addressing a deprecation warning. Future pandas will use the callable here directly, which is not what we want.